deploy/olm-catalog/argocd-operator/0.6.0/argoproj.io_argocds.yaml

```diff
@@ -537,10 +537,29 @@ spec:
                     description: Sharding contains the options for the Application
                       Controller sharding configuration.
                     properties:
+                      clustersPerShard:
```

Comment:
@ishitasequeira Are we supporting these new properties in older version (0.6.0) as well ?

Reply:
The file was automatically generated after executing `make bundle manifests`. I updated the file to not include these new properties.